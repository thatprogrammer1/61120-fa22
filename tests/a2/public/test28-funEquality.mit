
f = fun(){
  t = 5;
  return fun(x){ return x + t; };
};

g = f();
print("g(5) = " + g(5));
q = f();
h = g;

if(h == g){
}

if(g == f){
}
print("Function equality didn't crash.");
print("Bye!");
