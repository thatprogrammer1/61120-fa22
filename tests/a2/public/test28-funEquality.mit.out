g(5) = 10
Function equality didn't crash.
Bye!
